bigquery/deploy/README.md

```diff
@@ -46,19 +46,31 @@
     * In the project with the BigQuery dataset, make the service account a
     BigQuery Data Viewer.
   * Create cluster
-    * Go to https://console.cloud.google.com/kubernetes/list and click `Create Cluster`
-    * Change name to `elasticsearch-cluster`
-    * Set zone
-      * If you have [deployed the UI server and/or API server](https://github.com/DataBiosphere/data-explorer/tree/master/deploy),
-        select a zone in the same region as UI/API server. (Search for `Region`
-        in the App Engine Dashbaord.)
-      * If you not not yet deployed the UI/API servers, make a note what region
-        your cluster is in. Later when you run `gcloud app create`, select this
-        region. This is important because App Engine app regions cannot be
-        changed after `gcloud app create`. (The Elasticsearch deployement uses
-        [Internal Load Balancing](https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing),
-        so the API server will only be able to talk to Elasticsearch if it's in
-        the same region.)
+    * Run:
+    ```
+    gcloud container clusters create elasticsearch-cluster --num-nodes=1 --service-account=indexer@<project>.iam.gserviceaccount.com --zone=ZONE
```

Comment:
Yeah I agree it's better. I didn't allow parametrization since there are a lot of potential flags. Instead in the README, instructed users to manually update that script.